src/update.c

```diff
@@ -474,8 +474,8 @@ static enum swupd_code main_update()
 		  total_curl_sz);
 
 	if (server_version > current_version) {
-		printf("Update took %0.1f seconds, %ld bytes transferred\n", delta,
-		       total_curl_sz);
+		printf("Update took %0.1f seconds, %ld MB transferred\n", delta,
+		       total_curl_sz / 1000 / 1000);
```

Comment:
I always get confused in these cases if we should use the Base10 (1 MB = 1000000 bytes) or Base2 (1 MB = 1048576 bytes) for these conversions. What is more commonly used in Linux when referring to storage?

Reply:
1 MB = 10^6
1 MiB = 1024*1024

I used MB because that's what search uses.

Reply:
Both abbreviations are pretty commonly used these days, though sometimes `MB` (megabyte) is still used for cases when `MiB` (mebibyte) is technically the correct abbreviation.

Reply:
So, can I leave it the way it is?

Reply:
I am fine with using `MB` in this case, because as you said, swupd uses `MB` units elsewhere in the code.

Reply:
I guess if search is already using Base10 we should stick to it then, consistency is important